src/pages/learn/dm/Markdown_Tutorial/markdown.mdx

```diff
@@ -0,0 +1,257 @@
+---
+layout: ../../../../layouts/docs.astro
+---
+
+
+# __Markdown Tutorial Guide.__ 
+[@veritanuda](https://keybase.io/veritanuda)
+
+This guide will specifically explore the features of Markdown formatting throughout PlanarAlly. It is used in both [Notes](../../../../docs/game/notes) fields as well as the  [Chat](../../../../docs/game/chat) window. 
+
+Information in this guide was heavily sourced from the [Basic Syntax Guide](https://markdownguide.offshoot.io/basic-syntax/)
```

Comment:
Markdown comes in many flavours and some syntax might or might not be supported depending on some extra extensions etc.

I think it would be good to mention that PA uses markdown-it for it's markdown rendering, they have an interactive demo at https://markdown-it.github.io which will make it easier for users to play around with it and allows us to focus on things that are more PA specific.

The example in View and Edit mode you have in the Text Effects section is definitely nice to keep, but I would trim down a lot on the other sections that go over all the syntax, especially all the different heading sections and list options as those are relatively repetitive. (and the heading examples also do funky stuff with the navigation on the right)

The Images and Links section you can keep, but as I mentioned in the Chat section, there is additional support for URLs/Images in the chat window on top of the markdown syntax.

Reply:
Yeah.. was not sure which Markdown lib you were using. Thanks for the info and the link. I can reword it.